package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "fastify-passport",
+  "version": "0.0.1",
+  "description": "Simple, unobtrusive authentication for Fastify.",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "rm -rf ./dist && mkdir dist && tsc --outDir dist && git rev-parse HEAD > BUILD_SHA",
```

Comment:
Please add https://www.npmjs.com/package/rimraf to devDeps and here, to support also windows developers/CI.

Reply:
Isnt tsc taking care of cleaning dist folder?

Reply:
No, tsc never cleans, it just adds. For example, if you remove a ts file it will be still present in the dist folder.